public/app/core/components/TransformersUI/rowsToFields/rowsToFields.ts

```diff
@@ -0,0 +1,193 @@
+import { map } from 'rxjs/operators';
+import {
+  anyToNumber,
+  ArrayVector,
+  DataFrame,
+  DataTransformerID,
+  DataTransformerInfo,
+  Field,
+  FieldColorModeId,
+  FieldConfig,
+  FieldType,
+  getFieldDisplayName,
+  ReducerID,
+  ThresholdsMode,
+} from '@grafana/data';
+
+export interface RowToFieldsTransformOptions {
+  nameField?: string;
+  valueField?: string;
+  mappings?: RowToFieldsTransformMappings[];
```

Comment:
is there, or should there be a difference between empty array and undefined?  Perhaps empty means "don't map any config"?  and null will match all known configs?

Reply:
Why would you want to use this transform if you do not want to map anything?